src/Jackett/Indexers/FrenchTorrentDb.cs

```diff
@@ -0,0 +1,172 @@
+﻿using CsQuery;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace Jackett.Indexers
+{
+    class FrenchTorrentDb : IndexerInterface
+    {
+        public event Action<IndexerInterface, Newtonsoft.Json.Linq.JToken> OnSaveConfigurationRequested;
+
+        public event Action<IndexerInterface, string, Exception> OnResultParsingError;
+
+        class ConfigurationDataBasicLoginFrenchTorrentDb : ConfigurationData

```

Comment:
I briefly looked at this trackers login page and didn't see a captcha. Is there a reason you couldn't get a normal username/password login to work? It always sucks having to resort to using a cookie.